db/schema.rb

```diff
@@ -290,7 +275,6 @@
   end
 
   add_foreign_key "game_aliases", "games", on_delete: :cascade
-  add_foreign_key "google_users", "users"
```

Comment:
Thanks, this was from a branch-in-progress and it slipped into master accidentally.